javascripts/2048.js

```diff
@@ -1,32 +1,276 @@
 $(document).ready(function() {
   console.log('ready!');
+  arrayInitialize();
+  initializeGame();
   $('body').keydown(function(event){
     var arrow_keys = [37, 38, 39, 40];
+
     if(arrow_keys.indexOf(event.which) > -1) {
-      var tile = $('.tile');
-      moveTile(tile, event.which);
+      // var tile = $('.tile');
+      moveTile(event.which);
       event.preventDefault();
     }
   })
 })
 
-function moveTile(tile, direction) {
+function pop(tile) {
+  $(tile).addClass("popper")
+  .on("animationend",function() {
```

Comment:
When chaining functions, lining up code in a certain way can make it easier to read. Consider formatting your code this way:
```
  $(tile).addClass("popper").on("animationend", function() {
    $(this).removeClass("popper");
  });
```
or
```
  $(tile).addClass("popper")
    .on("animationend", function() {
      $(this).removeClass("popper");
    });
```